jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/AnnotationChecker.java

```diff
@@ -124,6 +130,35 @@ private static boolean checkMethodAnnotationsInClass(
     return true;
   }
 
+  // If the given method matches the expected test method name 'expectNullable', check if the method
+  // has the 'javax.annotation.Nullable' annotation on it exactly once.
```

Comment:
Can we make this a javadoc? I know it's a private method, but since we are already kind of doing a documentation comment...

Reply:
Done.